src/common/logger/logger.ts

```diff
@@ -23,6 +24,46 @@ export interface Logger {
   addPrefix: Unary<String, Logger>;
 }
 
+class JSONLogger implements Logger {
+
+  constructor(private logger = "turnilo") {
+  }
+
+  log(message: string, extra: Record<string, string>) {
+    console.log(JSON.stringify({
+      message,
+      time: new Date().toISOString(),
+      level: "INFO",
+      logger: this.logger,
+      ...extra
+    }));
+  }
+
+  addPrefix(prefix: string): Logger {
```

Comment:
I would remove this method at all, or at least rename it to something like `withLogger()`.